easybuild/easyconfigs/g/glew/glew-2.1.0-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,36 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+# th77embl updated to GCCcore 10.2.0
+easyblock = 'ConfigureMake'
+
+name = 'glew'
+version = '2.1.0'
+
+homepage = 'http://glew.sourceforge.net/'
+description = """The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source
+C/C++ extension loading library. GLEW provides efficient run-time mechanisms
+for determining which OpenGL extensions are supported on the target platform."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-%(version)s.tgz']
+checksums = ['04de91e7e6763039bc11940095cd9c7f880baba82196a7765f727ac05a993c95']
+
+builddependencies = [('binutils', '2.35')]
+
+dependencies = [('Mesa', '20.2.1')]
```

Comment:
This actually has a dependency on X11 as well (i see it link to libXau, libXdmcp, libxcb)